### PR TITLE
:construction_worker: use golang 12

### PR DESCRIPTION
**Pull request by manland**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -138,25 +149,17 @@ else
...
-	cd server && $(GO) test -race -v ./...
+	cd server && $(GO) test -race -v -coverprofile=coverage.txt ./...
 endif
 ifneq ($(HAS_WEBAPP),)
 	cd webapp && $(NPM) run fix;
 endif
 
-## Creates a coverage report for the server code.
-.PHONY: coverage
-coverage: server/.depensure webapp/.npminstall
```
Comment (hanzei) [inline Makefile:153]: Why did you remove this command?

Comment (manland) [reply Makefile:153]: I don't know i have just copy/paste from mattermost-plugin-gitlab which is a copy/paste of mattermost-plugin-github !!? Maybe I need to copy/paste from another plugin !!?? DO you have a base reference for makefile ?

Comment (hanzei) [reply Makefile:153]: Yes. Use https://github.com/mattermost/mattermost-plugin-sample/blob/master/Makefile. This is the "latest version" of the Makefile.

Comment (hanzei) [reply Makefile:153]: Please note that the sample plugin recently moved to go modules.
